#request

---
#response
float32[] cp_params
int8 real_time_track
bool success
